# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/service-workers/service-worker/update-not-allowed.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/service-workers/service-worker/update-not-allowed.https.html.ini
--- a/testing/web-platform/meta/service-workers/service-worker/update-not-allowed.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/service-workers/service-worker/update-not-allowed.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -2,19 +2,20 @@
   disabled:
     if os == "android": bug 1550895 (frequently fails on geckoview)
   expected:
+    if not fission and (os == "linux") and not debug and not webrender: ["ERROR", "TIMEOUT"]
     if fission and (os == "linux"): ["ERROR", "TIMEOUT"]
     ERROR
   [ServiceWorkerRegistration.update() from installing service worker throws.]
     expected:
+      if not fission and (os == "linux") and not debug and not webrender: ["PASS", "TIMEOUT"]
       if fission and (os == "linux"): ["PASS", "TIMEOUT"]
 
   [ServiceWorkerRegistration.update() from active service worker succeeds while installing service worker.]
     expected:
-      if (os == "linux") and not webrender and not debug and (processor == "x86"): ["PASS", "FAIL"]
-      if (os == "linux") and not webrender and not debug and (processor == "x86_64"): ["PASS", "TIMEOUT", "FAIL"]
+      if (os == "win") and (processor == "x86") and not debug: ["PASS", "FAIL"]
       if (os == "linux") and webrender and fission: ["PASS", "FAIL", "TIMEOUT", "NOTRUN"]
       if (os == "linux") and not webrender and debug: ["PASS", "FAIL"]
-      if (os == "win") and (processor == "x86") and not debug: ["PASS", "FAIL"]
+      if (os == "linux") and not webrender and not debug: ["PASS", "NOTRUN", "FAIL", "TIMEOUT"]
       if (os == "mac") and not debug: ["PASS", "TIMEOUT"]
 
   [ServiceWorkerRegistration.update() from client succeeds while installing service worker.]